app/views/layouts/admin.html.erb

```diff
@@ -61,6 +61,12 @@
               </a>
             </li>
             <%= render "admin/shared/nested_sidebar", menu_items: AdminMenu.navigation_items %>
+            <li class="block s:hidden">
```

Comment:
Small screens don't show the top-right buttons that allow creators to go back to the Forem's homepage. This is why this option is added only for small screens, so they are able to navigate back to the site.